posts/five-interns-walk-into-an-office.md

```diff
@@ -0,0 +1,44 @@
+# Five interns walk into an office...
+This blog represents the combined experiences of the five interns (Clement Dessoude, Yashar Atajiang, Auguste Lehuger, Spencer Hamilton, and Alexandre Andreini)  in Ippon’s Richmond office during the summer of 2018, including the projects we worked on, the technologies and skills we learned, and our experiences with the unique culture at Ippon. Some of us travelled from across the Atlantic to be at Ippon and some of us travelled just down the street. Some of us were here for half a year and some of us were here for half the summer. None of our experiences were exactly the same but we all got great work experience and exposure to new technologies while having some fun at the same time.
+## The Projects
```

Comment:
Start titles at level 1 (the first line is the title -> should be in the metadata)